toto/runlib.py

```diff
@@ -169,9 +165,8 @@ def create_link_metadata(name, materials, products, byproducts,
 
 def run_link(name, materials, products, toto_cmd_args, record_byproducts=False):
   """Performs all actions associated with toto run-link.
-  XXX: This should probably be atomic, i.e. all or nothing"""
+  XXX LP: This should probably be atomic, i.e. all or nothing"""
```

Comment:
I personaly prefer the FIXME/TODO macros, as they are highlighted automatically by most IDE's. Although it's just a personal preference.